### PR TITLE
orcid: make tests repeatable

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: backend/tests/integration/orcid/conftest.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import pytest
 
 IS_VCR_ENABLED = True
-IS_VCR_EPISODE_OR_ERROR = True  # False to record new cassettes.
+IS_VCR_EPISODE_OR_ERROR = False  # False to record new cassettes.
```
Comment (ammirate) [inline backend/tests/integration/orcid/conftest.py]: leftover?

Comment (drjova) [reply backend/tests/integration/orcid/conftest.py]: @ammirate updated

Comment (ammirate) [reply backend/tests/integration/orcid/conftest.py]: @drjova There are two tests failing because of `WorkAlreadyExistsException` :(

**Comment by drjova** [comment]
@ammirate fixed

**Pull request merged**